test/tests/pbs_smoketest.py

```diff
@@ -335,10 +335,16 @@ def test_finished_jobs(self):
         self.server.manager(MGR_CMD_SET, NODE, a, self.mom.shortname)
         a = {'job_history_enable': 'True'}
         self.server.manager(MGR_CMD_SET, SERVER, a)
-        a = {'Resource_List.ncpus': 2}
+        shell_path = os.path.join(self.server.pbs_conf['PBS_EXEC'],
+                                  "bin", "pbs_python")
+        a = {'Resource_List.ncpus': 2, ATTR_S: shell_path}
```

Comment:
If Shell_Path_List is needed for running the eat cpu script, then it might be better to set this attribute inside create_eatcpu_job(). What do you think?

Reply:
If i move this code to create_eatcpu_job() then i need to pass shell path to the method as well. 
Since i did not find a way to access PBS_CONF from Job class.

Reply:
You can use DshUtils.parse_pbs_conf in Job class to get pbs.conf